app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="activity_main_text_size_tittle">20sp</dimen>
+    <dimen name="activity_main_margin_tittle">20dp</dimen>
+    <dimen name="activity_main_margin_icon">20dp</dimen>
+    <dimen name="activity_main_size_icon">45dp</dimen>
+    <dimen name="activity_main_padding_top_screen">50dp</dimen>
+    <dimen name="activity_main_padding_end_screen">15dp</dimen>
+    <dimen name="activity_main_text_size_screen_operation">30sp</dimen>
+    <dimen name="activity_main_text_size_screen_result">40sp</dimen>
+    <dimen name="activity_main_padding_bottom_screen_result">10dp</dimen>
+    <dimen name="activity_main_margin_bottom_screen_result">40dp</dimen>
+    <dimen name="activity_main_height_button">0dp</dimen>
```

Comment:
the 0dp when it refers to a height or width should be in the XML, without a dimen res